src/utils/series-utils.js

```diff
@@ -71,41 +74,48 @@ function collectSeriesTypesInfo(children) {
  * Each next value of attr is equal to the previous value plus the difference
  * between attr0 and attr.
  * @param {Array} children Array of children.
- * @param {string} attr Attribute.
+ * @param {string} stackAttr Attribute to stack by.
+ * @param {string} clusterAttr Attribute to cluster by (optional).
  * @returns {Array} New array of children for the series.
  */
 export function getStackedData(children, attr) {
-  const childData = [];
-  let prevIndex = -1;
-  children.forEach((child, childIndex) => {
+  return children.reduce((accumulator, series, seriesIndex) => {
```

Comment:
I figured the forEach was unnecessary - there's a bunch of them that would probably be cleaner as reductions.